build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
     alias(libs.plugins.dokka)
     alias(libs.plugins.gradle.nexus.publish)
     alias(libs.plugins.kotlinx.atomicfu) apply false
+    kotlin("plugin.lombok") version "2.0.0" apply false
```

Comment:
Why not to use:
`alias(libs.plugins.lombok)`
?

Reply:
actually this is probably not needed, I'll remove it